vault/logical_system.go

```diff
@@ -40,6 +40,7 @@ func NewSystemBackend(core *Core, config *logical.BackendConfig) (logical.Backen
 				"audit/*",
 				"raw/*",
 				"rotate",
+				"config/*",
```

Comment:
I wonder if this really needs sudo. e.g. for `config/cors` it probably won't be needed. Maybe just sudo on the relevant auditing paths under `config/`?